lib/DDG/Spice/GooglePlus.pm

```diff
@@ -12,6 +12,8 @@ topics "everyday", "social";
 category "ids";
 attribution github => ['https://github.com/jagtalon','jagtalon'],
             twitter => ['http://twitter.com/juantalon','jagtalon'];
```

Comment:
Looks like you need to remove the `;` at the end of this line. It's throwing an error.